pkg/controllers/v1/mpi_job_controller.go

```diff
@@ -62,6 +62,7 @@ const (
 	configMountPath         = "/etc/mpi"
 	kubexecScriptName       = "kubexec.sh"
 	hostfileName            = "hostfile"
+	discoverHostsScriptName = "discover_hosts.sh"
```

Comment:
This assumes that the image contains this script, right?

Reply:
Right, mpi-operator can provide a *default* `discover_hosts.sh` to users and still open to `discover_hosts.sh` alternatives.